cx/spark/test/server_test.js

```diff
@@ -0,0 +1,52 @@
+const detectIntentToSparkMessage = require('../server.js').detectIntentToSparkMessage;
+const sparkToDetectIntent = require('../server.js').sparkToDetectIntent;
+const assert = require('assert');
+
+describe('sparkToDetectIntent()', () => {
+    const message = {
+        text: 'Test Text'
+    };
+
+    const dialogflowRequest = {
+        session: 'sessionPath',
```

Comment:
It might be better to use a session path which has the consistent format with the API spec.
https://cloud.google.com/dialogflow/cx/docs/reference/rest/v3beta1/projects.locations.agents.sessions/detectIntent#path-parameters